airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -92,9 +92,9 @@ public AirbyteConnectionStatus check(final JsonNode config) {
           .build();
 
       if (UploadingMethod.GCS.equals(uploadingMethod)) {
-        final AirbyteConnectionStatus airbyteConnectionStatus = checkGcsPermission(config);
-        if (Status.FAILED == airbyteConnectionStatus.getStatus()) {
-          return new AirbyteConnectionStatus().withStatus(Status.FAILED).withMessage(airbyteConnectionStatus.getMessage());
+        final AirbyteConnectionStatus status = checkGcsPermission(config);
```

Comment:
This does not look like a functional change to me. Looks like we are renaming `airbyteConnectionStatus` to `status`. Because this PR is so large, I suggest we keep only functional changes in it to simplify the job of reviewing it.